nodes/widgets/locales/en-US/ui_text.html

```diff
@@ -26,7 +26,23 @@ <h3>HTML Content</h3>
     <h3>Dynamic Properties (Inputs)</h3>
     <p>Any of the following can be appended to a <code>msg.</code> in order to override or set properties on this node at runtime.</p>
```

Comment:
This should have been updated for `msg.ui_update`

Reply:
Helpers were only added client-side

Reply:
Right you are